src/main/java/Task.java

```diff
@@ -0,0 +1,79 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+   // private int id;
+
+    /*
+    private char type;
+    private String from;
+    private String to;
+     */
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    public String getDescription() {
+        return description;
+    }
+    public void markDone() {
+        this.isDone = true;
+    }
+
+    public void markDel() {
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return "[" + (isDone ? "X" : " ") + "]"; // mark done task with X
+    }
+
+    public String getType() {
```

Comment:
In Task.java, lines 31 - 41, are the functions getType(), getFrom(), getTo() really necessary?